### ISSUE TITLE
missing davis/DAVIS/vallist.txt

### ISSUE THREAD

**Issue by arjung128** [xiaolonw/TimeCycle#25]
I am trying to test my trained network, but the script can't seem to find vallist.txt at the location it expects it to be (DATASET_FOLDER/davis/DAVIS/vallist.txt). 

I dowloaded Test-Dev 2017 and Test-Challenge 2017 and couldn't find the file in either of the two. Which of the two are we expected to use to test (to get the same results as in the paper)?

Any ideas? Thanks!

**Comment by LiUzHiAn** [comment]
@arjung128  Hi,

You should download the `DAVIS-2017-trainval-480p` and using the validation set of that instead. In my case, I found two files in folder `ImageSets/2017`, i.e. `train.txt` & `val.txt`. Then, the script `gendavis_vallist.py` will work well with that.
